etl/data_helpers.py

```diff
@@ -6,6 +6,8 @@
 
 from etl.paths import REFERENCE_DATASET
 
+REGIONS = ["World", "Asia", "Africa", "North America", "South America", "Europe", "Oceania"]
+
 
 def calculate_region_sums(df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
This function has a few issues:
1. Firstly, even if you specify a list of `REGIONS`, in the end it's going to include in the output dataframe as many regions as there are in the `countries_regions` table of the `reference` dataset. For example, even if `High-income countries` is not specified in `REGIONS` it will be added to your data.
2. I don't think it's a good idea to create the `World` aggregate. The reason why we create our own aggregates is because there may be different definitions of continents. But there is only one definition of the World. So, unless your dataset does not have data for `World` (which is unusual) I'd avoid creating our own aggregate.
We should remove this function and use the one in datautils (which also needs to be improved). Up to you if you want to use it in this case, but I just wanted to let you know these caveats.